src/shortestpaths/dijkstra.jl

```diff
@@ -47,6 +47,7 @@ function dijkstra_shortest_paths(
     dists[srcs] = zero(T)
     pathcounts[srcs] = 1
     closest_vertices = Vector{U}()  # Maintains vertices in order of distances from source
+    popped_out = Vector{Bool}(zeros(nvg))   # popped_out[u]=true if shortest distance from src to u is found
```

Comment:
@sbromberger 
These are the test using this PR
`
julia> g = loadgraph("mygraph.jgz")
{20000, 10000} undirected simple Int64 graph

julia> swg = SimpleWeightedGraph(g);

julia> sbc = @time betweenness_centrality(g);
 11.677448 seconds (1.95 M allocations: 19.394 GiB, 4.63% gc time)

julia> wbc = @time betweenness_centrality(swg);
 11.727869 seconds (2.75 M allocations: 19.469 GiB, 4.62% gc time)

`